tests/www/views/test_views.py

```diff
@@ -599,26 +599,35 @@ def test_invalid_dates(app, admin_client, url, content):
     assert re.search(content, resp.get_data().decode())
 
 
-@pytest.mark.parametrize("enabled, dags_count", [(False, 5), (True, 5)])
+@pytest.mark.parametrize("enabled", [False, True])
 @patch("airflow.utils.usage_data_collection.get_platform_info", return_value=("Linux", "x86_64"))
 @patch("airflow.utils.usage_data_collection.get_database_version", return_value="12.3")
 @patch("airflow.utils.usage_data_collection.get_database_name", return_value="postgres")
 @patch("airflow.utils.usage_data_collection.get_executor", return_value="SequentialExecutor")
 @patch("airflow.utils.usage_data_collection.get_python_version", return_value="3.8.5")
+@patch("airflow.utils.usage_data_collection.get_plugin_counts")
 def test_build_scarf_url(
-    get_platform_info,
```

Comment:
The order of these mocks were all inverted :)

Reply:
Whoops